website/docs/reference/notifications.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Notifications give your project members hints about interesting events happening in their projects.
+To see your notifications, click on the ring bell icon in the top-right corner of the screen.
+You will only see the notifications from the other project members without the events triggered by yourself.
+
+![Notifications](/img/notifications.png)
+
+From notifications dropdown you can:
+* navigate to the features or change requests the notification is for
+* mark all notifications as read
+* filter our unread notifications
+
+## Notification events
+
+The following events are supported in notifications:
+* Feature created
+* Feature enabled
+* Feature archived
+* Feature enabled in environment
+* Change request sent to review (sent only to project members who can approve the change request)
+* Change request approved
+* Change request applied
```

Comment:
I think this would be clearer in a more active voice. Plus, I have some questions about some of these.

```suggestion
The following actions will trigger notifications:
* Creating a feature
* Feature enabled # @kwasniew what is the difference between this and enabling a feature in an env?
* Archiving a feature
* Enabling a feature in an environment
* Submitting a change request (sent only to project members who can approve the change request)
* Any change request is approved (or is it only **your** change requests?)
* Any change request is applied (or is it only **your** change requests being applied?)
```

Reply:
It was a mistake to list feature enabled twice. I took your suggestions but the last two decided to start with a verb to have a consistent list. I also repeated the information that only your project events will trigger notifications.

Reply:
Ah, yes, good catch! That consistency works better 🙌🏼  But I'm not sure I understand the last two items: does everyone in that project get notifications when you approve/apply a change request or just the person who created the change reqest?

Reply:
@kwasniew Did you see this comments regarding the last two points?

Reply:
Every project member gets notification about approved/applied Change Request.

Reply:
The only exception was review notification that's why it got extra description